examples/quilkin-filter-example/src/main.rs

```diff
@@ -14,16 +14,27 @@
  * limitations under the License.
  */
 
+use greet::Greet as ProtoGreet;
 use quilkin::filters::prelude::*;
 
-use bytes::Bytes;
 use serde::{Deserialize, Serialize};
+use std::convert::TryFrom;
 
 #[derive(Serialize, Deserialize, Debug)]
 struct Config {
     greeting: String,
 }
 
+impl TryFrom<ProtoGreet> for Config {
+    type Error = ConvertProtoConfigError;
+
+    fn try_from(p: ProtoGreet) -> Result<Self, Self::Error> {
+        Ok(Config {
+            greeting: p.greeting,
+        })
+    }
+}
+
```

Comment:
Should/could we using `include_proto!` here instead? I tried it, and it wanted me to have tonic as a dependency, so I'm assuming that's why we avoided it in the example?

Reply:
I've no issue with using `tonic`, if you depend on `quilkin`, you already depend on `tonic` indirectly.

Reply:
Agreed! Got it to work! Update coming soon.